src/schemas/22-0994/schema.js

```diff
@@ -162,7 +162,7 @@ const schema = {
   required: [
     'privacyAgreementAccepted',
     'applicantFullName',
-    'appliedForVAEducationBenefits',
+    'appliedForVaEducationBenefits',
```

Comment:
Do you need to add this in here? It won't make it required that this value is true, it just requires that the value is defined.

Reply:
Good question!

I know that we do want an answer either way because It's used by the regional office to know whether or not they need to find an existing 22-1990 for the applicant.